ui/pages/confirmations/hooks/test-utils.js

```diff
@@ -96,6 +98,24 @@ export const HIGH_FEE_MARKET_ESTIMATE_RETURN_VALUE = {
   estimatedGasFeeTimeBounds: {},
 };
 
+export const generateUseMultichainSelectorRouter =
+  ({ shouldShowFiat = true } = {}) =>
+  (selector) => {
+    if (selector === getMultichainNativeCurrency) {
+      return EtherDenomination.ETH;
+    }
+    if (selector === getMultichainCurrentCurrency) {
+      return 'USD';
+    }
+    if (selector === getMultichainShouldShowFiat) {
+      return shouldShowFiat;
+    }
+    if (selector === getMultichainIsEvm) {
+      return true;
+    }
+    return undefined;
```

Comment:
Maybe we could use `geenrateUseSelectorRouter` here, to avoid duplicating those stubs, maybe something like this would be enough:

```ts
export const generateUseMultichainSelectorRouter = generateUseSelectorRouter;
```